ontopy/patch.py

```diff
@@ -2,11 +2,17 @@
 
 # pylint: disable=protected-access
 import types
+from typing import Any, Generator, Tuple
```

Comment:
It is slightly more efficient to 

```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from typing import Any, Generator, Tuple
```

and then change all annotations using these to strings. 

In Python 3.14 annotations are automatically converted to strings by the interpreter. But until then, we have to do it manually.